mappings/net/minecraft/block/entity/LootableContainerBlockEntity.mapping

```diff
@@ -6,14 +6,17 @@ CLASS net/minecraft/class_2621 net/minecraft/block/entity/LootableContainerBlock
 	METHOD method_11281 setInvStackList (Lnet/minecraft/class_2371;)V
 		ARG 1 list
 	METHOD method_11282 getInvStackList ()Lnet/minecraft/class_2371;
-	METHOD method_11283 deserializeLootTable (Lnet/minecraft/class_2487;)Z
+	METHOD method_11283 lootTableFromNbt (Lnet/minecraft/class_2487;)Z
+		ARG 1 nbt
 	METHOD method_11285 setLootTable (Lnet/minecraft/class_2960;J)V
 		ARG 1 id
 		ARG 2 seed
-	METHOD method_11286 serializeLootTable (Lnet/minecraft/class_2487;)Z
+	METHOD method_11286 lootTableToNbt (Lnet/minecraft/class_2487;)Z
+		ARG 1 nbt
```

Comment:
I did it for consistency with the other names. Serialize to what? Now it tells you. We didn't want to use serialize/deserialize because Minecraft uses a bunch of different serialization formats.

Reply:
Couldn't we find a name that doesn't require a second take every time you read it ?

Reply:
such as?

Reply:
Should be `loadLootTableFromNbt` and `saveLootTableToNbt` according to your (prospector's) rationale, but I'd prefer `loadLootTableFrom` and `saveLootTableTo` (see param types)

Reply:
ah yes, that one does pass nbt doesn't it